### DISCUSSION TITLE
Incredible, but support for visual studio 2019 would be nice

### DISCUSSION THREAD

**Discussion by Large-Guy** [community/community#13534] [Copilot Conversations] [answered]
I tried tabnine before this and that was quite impressive. but this is mind blowing. It is so cool. But I really want to use it in visual studio 2019.

**Comment by hsenag** [comment, marked as answer]
Hi, thanks for the feedback. I'm afraid that for technical reasons [we only support Visual Studio 2022 17.2 and above](https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-visual-studio#prerequisites), and it's unlikely we would start supporting older versions.

**Reply by KEYLEDGER** [reply]
Was there not a VS2019 extension at some point in the past?

**Reply by hsenag** [reply]
> Was there not a VS2019 extension at some point in the past?

Nope, not for Copilot.
